mod/file/start.php

```diff
@@ -479,11 +479,6 @@ function file_set_icon_file($hook, $type, $icon, $params) {
 			$metadata_name = 'smallthumb';
 			break;
 
-		case 'large' :
```

Comment:
why only remove `large`?

Reply:
behaviour of large is the same as default... other cases make different names